src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/induction/InductionEntityMapper.kt

```diff
@@ -89,6 +90,35 @@ abstract class InductionEntityMapper {
       conductedBy = inductionEntity.conductedBy,
       conductedByRole = inductionEntity.conductedByRole,
       completedDate = inductionEntity.completedDate,
+      note = noteEntity?.let { NoteMapper.fromEntityToDomain(noteEntity) },
+    )
+
+  fun fromEntityToDomain(
```

Comment:
Did you need to overload this method, given you add it as a nullable arg with a default value of null in the original copy of the method (line 70)
Or is this for something else?